### PR TITLE
Feat/es2015

### PR DESCRIPTION
**Pull request by arschmitz**
Creating PR for commenting

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.js]
+indent_style = space
+indent_size = 2
+
+[*.{diff,md}]
```
Comment (arschmitz) [inline .editorconfig]: We don't nor will we have diff files so no need to include `.diff`

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "0.12"
```
Comment (arschmitz) [inline .travis.yml]: We should just go ahead and jump to latest this is just for development anyway

File: Gruntfile.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
+var babel = require('rollup-plugin-babel');
```
Comment (arschmitz) [inline Gruntfile.js]: should run this file against jshint jscs

File: Gruntfile.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+module.exports = function(grunt) {
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    usebanner: {
+      taskName: {
+        options: {
+          position: 'top',
+          banner: '/*! <%= pkg.title || pkg.name %> - v<%= pkg.version %> - <%= grunt.template.today("yyyy-mm-dd") %>\n * <%= pkg.homepage %>\n *\n * Copyright (c) <%= grunt.template.today("yyyy") %> <%= pkg.author.name %>;\n * Licensed under the <%= pkg.license %> license */',
```
Comment (arschmitz) [inline Gruntfile.js]: break this into multiple lines

File: Gruntfile.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    },
+    rollup: {
+      options: {
+        format: 'es6',
+        plugins: [
+          babel({
+            exclude: 'node_modules/**'
+          })
+        ],
+        intro: " (function(window, document, exportName, undefined) { \n'use strict';",
```
Comment (arschmitz) [inline Gruntfile.js]: wrong quotes

File: Gruntfile.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+        outro: "})(window, document, 'Hammer');"
```
Comment (arschmitz) [inline Gruntfile.js]: wrong quotes

File: Gruntfile.js [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+          ]
+        }
+      }
+    },
+    jshint: {
+      options: {
+        jshintrc: true
+      },
+      build: {
+        src: ['hammer.js']
```
Comment (arschmitz) [inline Gruntfile.js]: this should run against the source and test files as well

Comment (arschmitz) [inline Gruntfile.js]: and also this file

File: Gruntfile.js [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+      }
+    },
+    jscs: {
+      src: ['src/**/*.js'],
```
Comment (arschmitz) [inline Gruntfile.js]: this should be run against the test files and this file as well

File: Gruntfile.js [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+      options: {
+        config: "./.jscsrc",
+        force: true
+      }
+    },
+    watch: {
+      scripts: {
+        files: ['src/**/*.js'],
```
Comment (arschmitz) [inline Gruntfile.js]: this should also observe the test files

**Comment by arjunkathuria** [comment]
@arschmitz addressed in #1001 .

**Pull request merged**